File: src/codes/Users_Manual/advanced/pulse_width_encoder.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = new ControllerBuilder().AddDevice(new AUTD3(Vector3d.Zero)).Open(Nop.Builder());
var table = new ushort[65536];
autd.Send(new PulseWidthEncoder(table));

File: src/codes/Users_Manual/advanced/pulse_width_encoder.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~#include<vector>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder({autd3::AUTD3(autd3::Vector3::Zero())}).open(autd3::link::Nop::builder());
autd.send(autd3::PulseWidthEncoder([](const auto& dev) {
  return [](const auto i) { return 0; };
}));
//~return 0; }